org.mwc.debrief.legacy/src/Debrief/Wrappers/TrackWrapper.java

```diff
@@ -974,7 +975,6 @@ else if (point instanceof SensorWrapper) {
 					existing = oldS;
 
 					// and append the data points

```

Comment:
Are you sure the next line can be deleted?

Reply:
Yes, append was not implemented for TacticalDataWrapper and implements Layer was removed.

Reply:
Hmm. Here's the behaviour.  When a new `Sensor` is being added, we check to see if there is already a sensor of that name.  If there is a sensor of that name we add the points from the new sensor to the existing one.

This PR loses the last bit of functionality - we aren't adding the sensor contact wrappers from the new sensor to the old one. We're ignoring them.